constraints/src/test/java/gov/nasa/jpl/aerie/constraints/tree/ASTTests.java

```diff
@@ -923,6 +923,31 @@ public void testExternalRealResource() {
     assertEquivalent(expected, result);
   }
 
+  @Test
+  public void testSpansAlias() {
+    final var simResults = new SimulationResults(
+        Interval.between(0, 20, SECONDS),
+        List.of(),
+        Map.of(),
+        Map.of()
+    );
+
+    final var spans = new Spans(
+        Interval.between(0, 1, SECONDS)
+    );
+
+    final var environment = new EvaluationEnvironment(
+        Map.of(),
+        Map.of("my spans", spans),
+        Map.of(),
+        Map.of()
+    );
+
+    final var result = new SpansAlias("my spans").evaluate(simResults, environment);
```

Comment:
I notice that `SpansAlias.evaluate` has three parameters, but its called here with only two. How are we able to do that?

Reply:
We have default overloads for `Expression.evaluate` in the Expression interface.

Reply:
Ah, missed that, thank you!